<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HChatRoomServerMain</class>
 <widget class="QWidget" name="HChatRoomServerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HChatRoomServerMain</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>8</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="serverAddressLabel">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>IP地址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="serverAddressList">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="serverPortLabel">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>监听端口：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="serverPortEdit">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="minimum">
       <number>66666</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="serverStartButton">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>开启服务</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="serverConnectCount">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="serverSendButton">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>399</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverSendEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>385</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="serverConnectMsgEdit">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>79</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="serverConnectClient">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>79</y>
     <width>171</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
selection-background-color: rgb(85, 85, 127);</string>
   </property>
   <item>
    <property name="text">
     <string>客户端上线列表</string>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
